samples/quickstart.js

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Google LLC
+// Copyright 2020 Google LLC
```

Comment:
The copyright dates should stay where they were.  They're supposed to recognize the date the file was originally created.

Reply:
When I refactored the bigquery repo, I ended up bumping the headers there as well. Just want to confirm keeping them as is before merging this PR.